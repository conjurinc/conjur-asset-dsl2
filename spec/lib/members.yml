- !group &developers developers

- !members
  role: !user alice
  members:
    - *developers
    - 
      role: !group ops
      admin: true
