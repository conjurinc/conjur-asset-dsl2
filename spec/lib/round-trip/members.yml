- !group &developers developers

- !members
  role: !group everyone
  members:
    - *developers
    - !group
      id: support
    - !group marketing
    - 
      role: !group ops
      admin: true
