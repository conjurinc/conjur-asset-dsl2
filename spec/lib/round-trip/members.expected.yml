---
- &1 !group
  id: developers
- !members
  role: !group
    id: everyone
  member:
  - !member
    role: *1
  - !member
    role: !group
      id: support
  - !member
    role: !group
      id: marketing
  - !member
    role: !group
      id: ops
    admin: true
